src/main/java/pl/lodz/p/edu/flanki/endpoints/EventEndpoint.java

```diff
@@ -0,0 +1,36 @@
+package pl.lodz.p.edu.flanki.endpoints;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.lodz.p.edu.flanki.dtos.EventDto;
+import pl.lodz.p.edu.flanki.entities.Event;
+import pl.lodz.p.edu.flanki.services.EventService;
+
+import java.util.List;
+
+@CrossOrigin(origins = "*")
+@RestController
+@RequestMapping("/")
+public class EventEndpoint {
+
+    private EventService eventService;
+
+    @Autowired
+    public EventEndpoint(EventService eventService) {
+        this.eventService = eventService;
+    }
+
+    @RequestMapping(method = RequestMethod.POST, path = "event/create")
```

Comment:
skoro wszystko jest tutaj o eventach to moze by wrzucić część ścieżki 'event' po prostu do request mapping?